src/utils/hooks/useGetReviewAssignment.tsx

```diff
@@ -87,7 +87,13 @@ const useGetReviewAssignment = ({ reviewerId, serialNumber }: UseGetReviewAssign
       setAssignment({
         id: currentAssignment.id,
         review: review
-          ? { id: review.id, status: review.status as ReviewStatus, stage: { id: 0, name: '' } }
+          ? {
+              id: review.id,
+              status: review.status as ReviewStatus,
+              stage: { id: 0, name: '' },
+              isLastLevel: false,
+              level: 0,
```

Comment:
I'm still confused about this, why are you adding those dummy values here? If the ReviewAssigned has review. Shouldn't get this info from the review?

Reply:
this is the old one, I didn't want to touch the query it was using. can do though